scripts/dev_settings.sh

```diff
@@ -32,50 +32,34 @@ if [ -z "$SQLALCHEMY_DATABASE_URI" ]; then
   export SQLALCHEMY_DATABASE_URI="sqlite:////tmp/questionnaire.db"
 fi
 
-if [ -z "$EQ_DYNAMODB_ENDPOINT" ]; then
```

Comment:
The same way the `$SQLALCHEMY_DATABASE_URI` was added back in, this should be kept to allow easy switching of storage when developing.